src/router/index.ts

```diff
@@ -5,102 +5,111 @@ import {authenticated, notAuthenticated} from "@/router/interceptors";
 Vue.use(VueRouter);
 
 const routes = [
-  {
```

Comment:
Are there any changes to this file? It's marked as rewritten but I didn't find changes to it from the first glance

Reply:
Going to consider this file was unchanged